packaging/linux/aur/PKGBUILD

```diff
@@ -1,7 +1,7 @@
 # Edit on github: https://github.com/LizardByte/Sunshine/tree/nightly/packaging/linux/aur/PKGBUILD
 # Reference: https://wiki.archlinux.org/title/PKGBUILD
 
-pkgname=@SUNSHINE_AUR_PKG@
+pkgname='sunshine'
 pkgver=@PROJECT_VERSION@@SUNSHINE_SUB_VERSION@
```

Comment:
`sub_version` possibly not needed, since we're not publishing anything except releases.

Reply:
Will keep this for now.